data/user_data_base.ps2

```diff
@@ -199,13 +210,28 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+$timeout = new-timespan -Minutes 5
+$sw = [diagnostics.stopwatch]::StartNew()
+
+### Wait until tags are set or it's longer than 5 minutes
+Do {
+    Write-Output "Waiting for tags to be added"
+} while ((-not (Check-AnyTagExists) ) -and ($sw.elapsed -lt $timeout))
```

Comment:
it's a `while less than 5 minutes`..  and is right,  I should change the comment I guess.